src/popup/popup.js

```diff
@@ -276,19 +288,46 @@ $('#choose-license').comboTree({
 });
 loadUserDefaults();
 
+async function loadStoredSearch() {
+  if (localStorage !== null) {
+    inputText = localStorage.getItem('title') ? localStorage.getItem('title') : '';
+    elements.inputField.value = inputText;
+
+    storeSearch.pageNo = localStorage.getItem('pageNo');
```

Comment:
What does this line do?

Reply:
Actually I was checking once more whether "title" exists in local storage or not.